tests/conftest.py

```diff
@@ -17,6 +24,78 @@ def asgi(request):
     return request.param
 
 
+class _SuiteUtils:
```

Comment:
you didn't remove the util file

Reply:
No, I was thinking to take one step at a time. The comment is misleading and alluding to that the removal has happened, I will update the comment.

Reply:
since there are test, we can maybe to that in this pr too, but as you prefer

Reply:
I'll just update the comment for now. Better keep shipping in small increments :shipit: